test/unit/metric_test.rb

```diff
@@ -88,9 +88,9 @@ def test_destroyable?
 
   test 'fill owner' do
     service = FactoryBot.create(:simple_service)
-    service_metric = FactoryBot.build(:metric, service: service)
+    service_metric = service.metrics.build(system_name: "meth1")
```

Comment:
This will test that metrics created the old way (with `service` instead of an `owner`) will automatically set `owner` before validation. Since Factory changed to use new style, we build directly which is IMO the correct way given that factory can change any time.